fitbenchmarking/fitbenchmarking
# Matlab Fitting

#### Description of Work

Fixes #798 

Still to do:
- [x] Add in documentation for matlab software 
- [x] Add in tests

#### Testing Instructions

1. Choose Matlab in the options and check results produced
2. Check that documentation makes sense

Function: Does the change do what it's supposed to?

Tests: Does it pass? Is there adequate coverage for new code?

Style: Is the coding style consistent? Is anything overly confusing?

Documentation: Is there a suitable change to documentation for this change?
